packages/catalog-process/src/services/readModelService.ts

```diff
@@ -715,6 +712,82 @@ export function readModelServiceBuilder(
     ): Promise<EServiceTemplate | undefined> {
       return getEServiceTemplate(eserviceTemplates, { "data.id": id });
     },
+    async getEServiceDescriptorDocuments(
+      eserviceId: EServiceId,
+      descriptorId: DescriptorId,
+      offset: number,
+      limit: number
+    ): Promise<ListResult<Document>> {
+      const pipeline = [
```

Comment:
Do we really need a query here? Why don't we take the documents directly from a queried by ids e-service descriptor?